<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartAndEnd</class>
 <widget class="QWidget" name="StartAndEnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>结束位置：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>起始位置：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>获取起始结束位置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
